### PR TITLE
[Doc] Add note for docker compose users about mounting volume for grafana prometheus config

### PR DESCRIPTION
**Pull request by smiraldr**
Adding note for docker compose users about mounting volume for grafana prometheus config directly from running ray container for automatic self configuration of ray metrics.

## Why are these changes needed?
Found an issue in docker compose accessing ray container config for grafana and prometheus from /tmp/ray/session_latest/metrics

### CODE REVIEW

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:181]: Can we add this as a troubleshooting tip and include this tip in both Prometheus and Grafana troubleshooting sections?

Something like 

>  **Troubleshooting: Loading Ray configurations with Docker Compose**
> In the Ray container, the symbolic link "/tmp/ray/session_latest/metrics" points to the latest active Ray session. However, Docker does not support the mounting of symbolic links on shared volumes and you may fail to load the Prometheus and Grafana configuration files and default dashboards.
>
>To fix this issue, employ an automated shell script for seamlessly transferring the necessary Grafana and Prometheus configurations and dashboards from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective paths for the containers, which contains the recommended configurations and default dashboards to initiate the Prometheus and Grafana servers.

Comment (smiraldr) [reply doc/source/cluster/metrics.md:181]: Sure

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:84]:
```suggestion
In the Ray container, the symbolic link "/tmp/ray/session_latest/metrics" points to the latest active Ray session. However, Docker does not support the mounting of symbolic links on shared volumes and you may fail to load the Prometheus configuration files.
```

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:86]:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective paths for the containers, which contains the recommended configurations to initiate the Prometheus servers.
```

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:232]:
```suggestion
In the Ray container, the symbolic link "/tmp/ray/session_latest/metrics" points to the latest active Ray session. However, Docker does not support the mounting of symbolic links on shared volumes and you may fail to load the Grafana configuration files and default dashboards.
```

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:234]:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the necessary Grafana configurations and dashboards from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective path for the container, which contains the recommended configurations and default dashboards to initiate Grafana servers.
```

File: doc/source/cluster/metrics.md
Comment (scottsun94) [inline doc/source/cluster/metrics.md:86]:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```

**Comment by scottsun94** [comment]
@rkooo567 @angelinalg to take a look

**Review by scottsun94 — APPROVED** [review]

**Review by rkooo567 — APPROVED** [review]

**Review by architkulkarni — APPROVED** [review]

**Review by architkulkarni — CHANGES_REQUESTED** [review]
@smiraldr Can you check this test failure? It looks potentially related (warnings are treated as errors for this test)

https://buildkite.com/ray-project/oss-ci-build-pr/builds/28332#018946fe-b466-434f-86d6-d7b1370779f5/1855-2288

**Comment by smiraldr** [comment]
sure let me fix this.

File: doc/source/cluster/metrics.md
Comment (angelinalg) [inline doc/source/cluster/metrics.md:86]:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, the shared volume mount the shared volume on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```

File: doc/source/cluster/metrics.md
Comment (angelinalg) [inline doc/source/cluster/metrics.md:234]:
```suggestion
To fix this issue, employ an automated shell script for seamlessly transferring the necessary Grafana configurations and dashboards from the Ray container to a shared volume. To ensure a proper setup, mount the shared volume on the respective path for the container, which contains the recommended configurations and default dashboards to initiate Grafana servers.
```

**Review by angelinalg — APPROVED** [review]
Just a couple of nits around active voice. If you could fix those (I made a suggestion, but not sure if it's still technically correct), I'd appreciate it. Thanks!

File: doc/source/cluster/metrics.md
Comment (smiraldr) [reply doc/source/cluster/metrics.md:86]: I think it should be
```
To fix this issue, employ an automated shell script for seamlessly transferring the Prometheus configurations from the Ray container to a shared volume. To ensure a proper setup, mount the shared volume on the respective path for the container, which contains the recommended configurations to initiate the Prometheus servers.
```
There's an extra "the shared volume" before the newly suggested text
would help if you could change the suggestion , ill commit it.

**Comment by architkulkarni** [comment]
@smiraldr According to "bk://:book: Documentation" line on https://flaky-tests.ray.io/ it seems like the test is passing on master.  Can you try merging the latest master?  I agree the error is confusing, but if it still happens after merging master I think it's likely related to this change somehow.

**Comment by architkulkarni** [comment]
@maxpumperla I think we might need your expertise here, any idea how these headings could be triggering the `myst` error?

File: doc/source/cluster/metrics.md
Comment (architkulkarni) [inline doc/source/cluster/metrics.md:217]: Thanks for merging master again, it looks like the same test failed again.  It's probably due to some other issue in the file, but I have no idea what it is and we can't merge the PR with the test failing.  Because the error involves "header", I'm going to suggest using `** **` instead of `#` headers on the changed lines as a workaround, just to see if the error goes away so we can unblock this PR... do you mind giving it a shot?

Comment (smiraldr) [reply doc/source/cluster/metrics.md:217]: @architkulkarni this fixed the issue ! There are stil some other tests failing though , one looks like a timeout issue and other is validating url redirections. I think they are unrelated to this commit. Can you check and confirm once as well?

**Comment by architkulkarni** [comment]
Linkcheck failure https://github.com/aws/amazon-sagemaker-examples/tree/main/reinforcement_learning/rl_roboschool_ray unrelated
Ray train GPU doctest failure unrelated

Merging!

**Review by architkulkarni — APPROVED** [review]

**Pull request merged**
